OpenEdXMobile/src/main/java/org/edx/mobile/view/MyCoursesListFragment.java

```diff
@@ -289,6 +349,22 @@ protected void onRevisit() {
     public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
         super.onCreateOptionsMenu(menu, inflater);
         inflater.inflate(R.menu.my_courses, menu);
+        final Config config = environment.getConfig();
+        if (config.isTabsLayoutEnabled()) {
+            menu.findItem(R.id.menu_item_search).setVisible(false);
+            menu.findItem(R.id.menu_item_account).setVisible(true);
+            menu.findItem(R.id.menu_item_account).setIcon(
+                    new IconDrawable(getContext(), FontAwesomeIcons.fa_gear)
+                            .colorRes(getContext(), R.color.white)
+                            .actionBarSize(getContext()));
+        } else {
+            menu.findItem(R.id.menu_item_account).setVisible(false);
+            if (config.getCourseDiscoveryConfig().isCourseDiscoveryEnabled()) {
+                menu.findItem(R.id.menu_item_search).setVisible(true);
+            } else {
+                menu.findItem(R.id.menu_item_search).setVisible(false);
```

Comment:
This if-else block can be simplified to:

`menu.findItem(R.id.menu_item_search).setVisible(config.getCourseDiscoveryConfig().isCourseDiscoveryEnabled())`